### PR TITLE
Replace @rsuite/icons with better solution

### PR DESCRIPTION
**Pull request by antkiewiczk**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -178,6 +178,32 @@ To give further control over the content of HTML block, whitelisting certain tag
...
 
+### Icon Library
+
+The icon library used throughout the editor is react-icons. https://react-icons.github.io/react-icons
+
+It's a great library created reasonably long time ago, with a good support, that includes few of the most widely used icon packages, all for free.
+For consistency, we decided to only use one of the icon sets in our project, and we chose material-icons from Google.
+- it contains lots of icons
+- it utilizes ES6 imports that allows us to include only the icons that we are using in our project
+- it's free
```
Comment (michael-scheurer) [inline docs/README.md:189]: Does free mean open source?

File: docs/README.md [hunk 2]
```diff
@@ -178,6 +178,32 @@ To give further control over the content of HTML block, whitelisting certain tag
 
 We have, however, to be aware that it's almost impossible to be 100% sure that none of the code displayed as custom HTML is dangerous.
 
+### Icon Library
```
Comment (michael-scheurer) [inline docs/README.md:181]: Documentation is great!

File: packages/editor/src/client/api/index.ts [hunk 3]
```diff
@@ -117,7 +117,7 @@ export type ArticleRevision = {
   canonicalUrl?: Maybe<Scalars['String']>;
   url: Scalars['String'];
   image?: Maybe<Image>;
-  authors: Array<Maybe<Author>>;
```
Comment (michael-scheurer) [inline packages/editor/src/client/api/index.ts:120]: We shouldn't have any changes in this file, shouldn't we?

**Review by michael-scheurer — CHANGES_REQUESTED** [review]
Thanks for implementing. Looks super good. Before approving this PR, I would like to have a look in a deployed version.

File: packages/editor/src/client/api/index.ts (hunk 3 above)
Comment (Itrulia) [reply packages/editor/src/client/api/index.ts]: The 8.1.0 release was missing the generation of these files which is the reason why

**Review by Itrulia — APPROVED** [review]

**Review by michael-scheurer — APPROVED** [review]

**Pull request merged**